Tests/Contoso.Forms.Test.Strings/Strings.cs

```diff
@@ -58,7 +58,7 @@ public static class  TestStrings
         public static readonly string FailedToSendErrorReportLabel = "FailedToSendErrorReportLabel";
         public static readonly string ShouldProcessErrorReportLabel = "ShouldProcessErrorReportLabel";
         public static readonly string ShouldAwaitUserConfirmationLabel = "ShouldAwaitUserConfirmationLabel";
-        public static readonly string GetErrorAttachmentLabel = "GetErrorAttachmentLabel";
+        //public static readonly string GetErrorAttachmentLabel = "GetErrorAttachmentLabel";
```

Comment:
Should try to reduce to a minimum what we comment out, will be harder to restore the feature after otherwise.